docs/api/api.md

```diff
@@ -5,26 +5,36 @@
 
 The Mbed OS RTOS capabilities include managing objects such as threads, synchronization objects and timers. It also provides interfaces for attaching an application-specific idle hook function, reads the OS tick count and implements functionality to report RTOS errors.
 
+Most of the RTOS APIs are also compatible with Mbed OS bare metal profile. The Mbed OS bare metal profile is a compact profile of Mbed OS without an RTOS.
```

Comment:
I'd just say "Most of the Mbed OS APIs are also supported with the Mbed OS bare metal profile."